static/bidder-params/tappx.json

```diff
@@ -12,13 +12,29 @@
 			"type": "string",
 			"description": "An ID which identifies the adunit"
 		},
+		"mktag": {
+			"type": "string",
+			"description": "Minimum bid for this impression expressed in CPM (USD)"
+		},
 		"endpoint": {
 			"type": "string",
 			"description": "Endpoint provided to publisher"
 		},
 		"bidfloor": {
 			"type": "number",
 			"description": "Minimum bid for this impression expressed in CPM (USD)"
+		},
+		"rewarded": {
+			"type": "number",
```

Comment:
Please consider changing the type to an integer, as a number can also be floating point or scientific notation. I also recommend adding a min and max constraint in the schema:
```
  "minimum": 0,
  "maximum": 1,
```

Reply:
Hi! We resolve this in other comment. We now find useless to use the rewarded tappx param. We think that is better to tell to our publishers to send in video ext param directly like in all our oRTB connections. If the data is correct or incorrect we check in our side! 

We clean all this in a2ddc1f commit.